Makefile

```diff
@@ -18,6 +18,8 @@ HTTP_PROXY := "$(http_proxy)"
 HTTPS_PROXY := "$(https_proxy)"
 OVMS_MEDIA_DOCKER_IMAGE ?= mediapipe_ovms
 OVMS_MEDIA_IMAGE_TAG ?= latest
+BASE_IMAGE := "ubuntu:20.04"
```

Comment:
does it work in u22.04?

Reply:
No, needs another task to update and make it compile/work.